docs/DeveloperGuide.md

```diff
@@ -397,12 +397,17 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* * *`  | user                                       | see the profile of a particular student           | get their information                                        |
 | `* * *`  | user                                       | remove a student's profile                        | keep track of only students that I teach                     |
 | `* * *`  | user                                       | edit the details of a student                     | keep track of up-to-date information                         |
-| `* * *`  | user                                       | add individual tuition sessions                   | keep track of my tuition sessions                            |
-| `* * *`  | user                                       | delete an individual tuition session              | update my tuition schedule                                   |
+| `* * *`  | busy user                                  | add an individual tuition sessions                | save time by creating multiple sessions in one command       |
+| `* * *`  | user                                       | add a recurring tuition session                   | keep track of my tuition sessions                            |
+| `* * *`  | user                                       | delete a particular tuition session               | update my tuition schedule                                   |
+| `* * *`  | user                                       | delete the entire recurring session               | update my tuition schedule                                   |
+| `* * *`  | user                                       | delete a single session from a recurring session  | take a break from lessons once in a while                    |
```

Comment:
i think maybe: 'so that I can update cancelled tuition session'? haha

Reply:
Ok! Sounds better! I have updated it in the latest commit! Thanks!